### DISCUSSION TITLE
error in importing file

### DISCUSSION THREAD

**Discussion by gippo2gippo** [ente-io/ente#3812] [Q&A]
In importing the data file exported with the app in my iPhone, on the app on my notebook with macOS I get an error:
"could not parse the selected file"

Is it a bug?

Thanks for your help.

  Gian Paolo

**Comment by crKRoEYS** [comment]
Instructions:
https://help.ente.io/photos/migration/from-apple-photos/

"If you run into any issues during this migration, please reach out to support@ente.io and we will be happy to help you!"
